marklogic-data-hub/src/main/resources/ml-modules/root/marklogic-data-hub-spark-connector/readRows.sjs

```diff
@@ -27,4 +42,6 @@ if (endpointState.batchNumber <= partition.batchCount) {
   endpointState.batchNumber = endpointState.batchNumber + 1;
 }
 
+console.log("results", xdmp.toJsonString(results));
```

Comment:
Can you remove this line of logging? I may have accidentally included it in the PR I submitted.